main.js

```diff
@@ -5,16 +5,35 @@
 'use strict'
 
 const { getPrepareStackTrace } = require('./js/stack-trace/')
+const { cacheRewrittenSourceMap } = require('./js/source-map')
 
 class DummyRewriter {
   rewrite (code, file) {
     return code
   }
 }
 
+let NativeRewriter
+class CacheRewriter {
+  constructor () {
```

Comment:
You must pass config options from `CacheRewriter` constructor to `NativeRewriter` constructor

```suggestion
  constructor (config) {
    if (NativeRewriter) {
      this.nativeRewriter = new NativeRewriter(config)
    } else {
      this.nativeRewriter = new DummyRewriter()
    }
    ...
  }
  }
```